src/lib/core/CHIPConfig.h

```diff
@@ -2690,6 +2690,15 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_SESSION_RELEASE_DELEGATES 2
 #endif
 
+/**
+ * @def CHIP_CONFIG_MAX_SESSION_RECOVERY_DELEGATES
+ *
+ * @brief Defines the max number of SessionRecoveryDelegate
+ */
+#ifndef CHIP_CONFIG_MAX_SESSION_RECOVERY_DELEGATES
+#define CHIP_CONFIG_MAX_SESSION_RECOVERY_DELEGATES 2
```

Comment:
Can we set it to 3 or 4 by default ? I'm working on updating `chip-tool` to have 3 instances of the device controller at the same time and I will need to update that otherwise :)

Reply:
Ok